pkg/util/util.go

```diff
@@ -239,3 +244,147 @@ func SplitQualifiedName(str string) (string, string) {
 func JoinQualifiedName(namespace, name string) string {
 	return path.Join(namespace, name)
 }
+
+type Route struct {
+	Interface   string
+	Destination net.IP
+	Gateway     net.IP
+	// TODO: add more fields here if needed
+}
+
+func getRoutes(input string) ([]Route, error) {
```

Comment:
the reason this originally took a Reader was so that it could be tested easily by injecting a string reader.  Please follow that model.  The caller should open the file and pass the reader in to here.